ami/flowchart/library/Roi.py

```diff
@@ -30,6 +31,7 @@ def __call__(self, img, mask=None):
                 logger.info('update hpolar with cx:%.1f, cy:%.1f, ro:%d, ri:%d, ao:%.1f, ai:%.1f, nr:%d, na:%d' %
                             (cx, cy, ro, ri, ao, ai, nr, na))
                 hp = self.hpolar = ur.polar_histogram(img.shape, mask, cx, cy, ro, ri, ao, ao+ai, nr, na)
```

Comment:
Should we clear this weird syntax here? Or is it what we want?

Reply:
I think its ok.